event-based-simulators/main/cumulocityAPI.py

```diff
@@ -1,25 +1,34 @@
 import json, os, logging, requests, base64
 
-C8Y_BASE = os.environ.get('C8Y_BASEURL') or 'http://localhost:8080'
-C8Y_TENANT = os.environ.get('C8Y_TENANT') or 't100'
-C8Y_USER = os.environ.get('C8Y_USER') or 'test'
-C8Y_PASSWORD = os.environ.get('C8Y_PASSWORD') or 'test'
+C8Y_BASE = 'http://localhost:8080'
```

Comment:
Deleting os.environ.get('C8Y_BASEURL') does not seem right. I think the the simulator uses this to get the credentials in c8y microservice environment

Reply:
The same for the other variables

Reply:
Sorry, It's my mistake during the change for the test